_data/conference.yaml

```diff
@@ -1,44 +1,43 @@
-  slides: https://github.com/ansible-middleware/wfl-mini-conf-demo/blob/main/slides.md
-- title: Break
+    - "Abstract to follow."
+  speaker: bstansberry
+  bio: "Brian Stansberry is the lead of the WildFly application server project and the principal architect of Red Hat's JBoss Enterprise Application Platform."
+- title: Lightning Talk 2
   from: "15:45"
   to: "16:00"
-- title: What is new in WildFly and roadmap for future releases
+- title: WildFly Remediation with Event-Driven Ansible
   from: "16:00"
   to: "16:45"
   abstract:
-    - "In this session we'll provide an overview of what's new in the WildFly project in the last several months, particularly in the WildFly 31 release. Then we'll describe where we see things going in WildFly over the remainder of 2024 -- release plans, a possible roadmap for Jakarta EE 11 support, plans around Java SE support and ideas around significant new features and initiatives."
-  speaker: bstansberry
-  bio: "Brian Stansberry is the lead of the WildFly application server project and the principal architect of Red Hat's JBoss Enterprise Application Platform."
-  slides: https://www.wildfly.org/assets/data/wmc_whats_new_and_roadmap.pdf
-- title: WildFly Glow
+    - "Event Driven Ansible is new capacity of the automation tool to react to change or incidents on a system. A basic application of this would be for instance to detect an out-of-memory error in an app hosted by Wildfly and trigger a restart. We'll demo that during the presentation."
+  speaker: rpelisse
+  bio: "Romain Pelisse works at Red Hat for over a decade. He started as runtimes consultant, building on expertise on JBoss EAP (Wildfly), and moved to engineering where he became the lead of the Ansible runtimes initiative, focusing on providing the best integration possible between Red Hat middleware solutions and Ansible Automation Platform."
+- title: Lightning Talk 2
   from: "16:45"
-  to: "17:30"
+  to: "17:00"
+- title: "Let's be smart: Doing AI with WildFly"
+  from: "17:00"
+  to: "17:40"
   abstract:
-    - "In this session you will be introduced to WildFly Glow, a new approach to the WildFly provisioning."
-    - "We will cover WildFly provisioning in general, discuss potential issues, and we will see how WildFly Glow simplifies the WildFly provisioning experience."
-    - "During the presentation WildFly Glow tooling will be demonstrated."
-  speaker: jfdenise
-  bio: "Jean-François Denise is a principal software engineer at Red Hat. He is involved in WildFly provisioning and tooling (WildFly CLI, WildFly Bootable JAR, Openshift images, WildFly Glow)."
-  slides: https://www.wildfly.org/assets/data/wmc_glow.pdf
+    - "A sample on how to use the new AI feature pack to build an AI-powered application."
```

Comment:
Learn how to use the new WildFly AI Feature Pack to build an AI-powered application.In this presentation we will see how we can configure WildFly to a Large Language Model and build a Retrieval Augmented Application using CDI to inject all the pieces we need.